src/components/common/TmSessionExplore.vue

```diff
@@ -96,6 +96,11 @@ import TmFormMsg from "common/TmFormMsg"
 import bech32 from "bech32"
 import { formatAddress } from "src/filters"
 
+const isPolkadotAddress = address => {
```

Comment:
lol, all the logic in this file should move out. instead address format errors should be bubbled up from the common function in the store (connection.js > getNetworkByAccount)

Reply:
@Bitcoinera why are you doing this on this PR? was it causing an issue here?

Reply:
I saw the bug on the Kanban and I am wondering just like you why this `this.isPolkadotAddress` is here 🤷‍♀ 

Very odd

Reply:
I really like this PR's name btw 😸